_includes/archive-single.html

```diff
@@ -36,6 +36,23 @@ <h2 class="archive__item-title" itemprop="headline">
     {% if post.read_time %}
       <p class="page__meta"><i class="far fa-clock" aria-hidden="true"></i> {% include read-time.html %}</p>
     {% endif %}
-    {% if post.excerpt %}<p class="archive__item-excerpt" itemprop="description">{{ post.excerpt | markdownify | strip_html | truncate: 160 }}</p>{% endif %}
+    {% if post.collection == "book_reports" %}
+      <hr />
+      
+      {% comment %}
+        Split the post's excerpt into two parts using the unordered list
```

Comment:
Line is breaking a little too early. Line limit is <= 80 characters, so keeping " element" on the same line makes it exactly 80 characters.

Reply:
Hmm, weird it shows as 81 characters in my editor with "element" on the first line?  I updated it so it is on the first line now though in aacec1cfb4e333c43d9cfb15464fc84077068c10.